dashboard/src/components/forms/DropdownButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement, useState } from "react";
+import Link from "next/link";
+
+interface DropdownButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  onClick?: () => void;
+  href?: string;
+  icon?: JSX.Element;
+}
+
+export const DropdownButton: React.FC<DropdownButtonProps> = ({
+  onClick = () => undefined,
```

Comment:
Why have this value if it's just going to be undefined?

Reply:
That's a default value

Reply:
As i mentioned above you're already extending `React.ButtonHTMLAttributes<HTMLButtonElement>` so instead of providing a default just access the value passed down from `HTMLButtonElement`

```suggestion
  onClick,
```